docs/openapi-issuer.yaml

```diff
@@ -0,0 +1,627 @@
+      tags:
+        - Signer
+      summary: Get Signer By Fiscal COde
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/GetSignerByFiscalCodeBody"
+      responses:
+        "201":
+          description: The Signer detail
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SignerDetailView"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "429":
+          $ref: "#/components/responses/TooManyRequests"
+        default:
+          $ref: "#/components/responses/Unexpected"
+
+  /signature-requests:
+    post:
+      operationId: requestSignature
+      tags:
+        - Signature Request
+      summary: Request a Signature
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/RequestSignatureBody"
+      responses:
+        "201":
+          description: The Signature Request detail
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SignatureRequestDetailView"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "429":
+          $ref: "#/components/responses/TooManyRequests"
+        default:
+          $ref: "#/components/responses/Unexpected"
+
+  /signature-requests/{id}:
+    get:
+      operationId: getSignatureRequest
+      tags:
+        - Signature Request
+      summary: Get a Signature Request by Id
+      parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+            $ref: "#/components/schemas/Id"
+      responses:
+        "200":
+          description: The Signature Request detail
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SignatureRequestDetailView"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "429":
+          $ref: "#/components/responses/TooManyRequests"
+        default:
+          $ref: "#/components/responses/Unexpected"
+
+  /signature-requests/{id}/status:
+    put:
+      operationId: putSignatureRequestStatus
+      tags:
+        - Signature Request
+      summary: Set the status of a Signature Request
+      parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+            $ref: "#/components/schemas/Id"
```

Comment:
there is no requestBody. How can I specify the status?

Reply:
good catch! :D